pkgs/development/python-modules/llfuse/default.nix

```diff
@@ -31,8 +31,13 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [ contextlib2 ];
 
+  preConfigure = ''
```

Comment:
```suggestion
  postPatch = ''
```

Reply:
I feel the same here as in https://github.com/NixOS/nixpkgs/pull/145072#discussion_r746097535.  What do you think?